examples/post-sync/README

```diff
@@ -0,0 +1,135 @@
+# Sync Status Watch Controller
+
+A Kubernetes controller that monitors and reports sync error status for Config Sync's `RootSync` and `RepoSync` resources.
+
+## Prerequisites
+
+- Go 1.24+
+- Docker
+- kubectl
+- Access to a Kubernetes cluster with Config Sync installed
+- Kubernetes cluster (v1.20+ recommended)
+- Container registry access (public or private)
+
+### **Note**
+⚠️ **This component is not released with Config Sync**.  
+- It is **only compatible** with the **Config Sync API at `main` branch HEAD**.  
+- **Backward compatibility is not supported at this moment**.
+- **Version management and continuous builds** of this controller **must be handled by the user**.  
+
+## Image Registry Setup
+
+### Using Public Registry (Example: GAR)
+This guide uses Google Artifact Registry (GAR) as an example public registry:
+
+### Using Google Artifact Registry (GAR)
+1. **Enable Artifact Registry API**:
+   ```sh
+   gcloud services enable artifactregistry.googleapis.com
+   ```
+
+2. **Create a GAR Repository** (if not exists):
+   ```sh
+   gcloud artifacts repositories create sync-status-watch \
+     --repository-format=docker \
+     --location=REGION (e.g., us-central1) \
+     --description="Docker repository for Sync Status Watch"
+   ```
+
+3. **Configure Docker Authentication**:
+   ```sh
+   gcloud auth configure-docker REGION-docker.pkg.dev
+   ```
+
+*Note: For production environments, consider using a private registry for better security and access control.*
+
+### Alternative Registry Options
+- **Private Registry**: Use private container registry
+- **Local Registry**: For development/testing with `minikube` or `kind`
+- **Other Public Registries**: Docker Hub, GitHub Container Registry
+
+## Building the Container
```

Comment:
This doesn't feel very user friendly. Can we create a makefile with some simple targets?

e.g. `make build push deploy`

Reply:
Done, added a Makefile and updated guide